projects/igniteui-angular/src/lib/grids/tree-grid/tree-grid.component.spec.ts

```diff
@@ -0,0 +1,148 @@
+import { Component, ViewChild } from '@angular/core';
+import { async, TestBed, fakeAsync, tick } from '@angular/core/testing';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { IgxTreeGridModule } from './index';
+import { IgxTreeGridComponent } from './tree-grid.component';
+import { DisplayDensity } from '../../core/displayDensity';
+import { configureTestSuite } from '../../test-utils/configure-suite';
+import { SampleTestData } from '../../test-utils/sample-test-data.spec';
+import { By } from '@angular/platform-browser';
+
+describe('IgxTreeGrid Component Tests', () => {
+
+    describe('IgxTreeGrid - default rendering for rows and columns', () => {
+        configureTestSuite();
+        let fix;
+        let grid: IgxTreeGridComponent;
+
+        beforeEach(async(() => {
+            TestBed.configureTestingModule({
+                declarations: [
+                    IgxTreeGridWrappedInContComponent
+                ],
+                imports: [
+                    NoopAnimationsModule, IgxTreeGridModule]
+            }).compileComponents();
+        }));
+
+        it('should match width and height of parent container when width/height are set in %', fakeAsync(() => {
+            fix = TestBed.createComponent(IgxTreeGridWrappedInContComponent);
```

Comment:
Extract the component creation in a single beforeEach() as all tests are using the same component.